### PR TITLE
Add menu bar to remaining editor types.

### LINKED ISSUE #16571: Make menu bar available in each editor type
## 📝 Provide a description of the new feature

For now, we have integrated the menu bar only in the classic, document, and multi-root editors. There is no reason why other editor types should not have the possibility to display the menu bar. Additionally, it will simplify the whole logic.

Task requirements:
1. Ensure that all editor types have the possibility to display the menu bar.
2. Refactor the code to simplify it by moving the logic responsible for registering the menu bar to `EditorUI`.

---

If you'd like to see this feature implemented, add a 👍 reaction to this post.

### PR DESCRIPTION
**Pull request by mremiszewski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Feature (editor-balloon, editor-inline): Add menu bar support. Closes #16571.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

**Review by DawidKossowski — CHANGES_REQUESTED** [review]
@mremiszewski, In the case of the balloon editor, we cannot include the menu bar inside the balloon as it breaks the UX. Instead, we should handle it similarly to the multi-root editor. You should provide an independent element that integrators can render in any place. After making this change, moving the entire logic to `EditorUI` should be possible.

File: packages/ckeditor5-ui/src/editorui/editorui.ts [hunk 1]
```diff
@@ -322,6 +324,34 @@ export default abstract class EditorUI extends /* #__PURE__ */ ObservableMixin()
...
+				cancel();
+			}
+		} );
+
+		this.editor.keystrokes.set( 'Alt+F9', ( data, cancel ) => {
+			if ( !menuBarViewElement.contains( this.editor.ui.focusTracker.focusedElement ) ) {
+				menuBarView!.focus();
+				cancel();
+			}
+		} );
```
Comment (Mati365) [inline packages/ckeditor5-ui/src/editorui/editorui.ts:352]: This handler has to be extended after merging this PR: https://github.com/ckeditor/ckeditor5/pull/16573, we added focus bordering flag assign on alt+f9.

**Review by DawidKossowski — COMMENTED** [review]
Just one comment. Besides that LGTM.

File: packages/ckeditor5-ui/src/editorui/editorui.ts (hunk 1 above)
Comment (Mati365) [reply packages/ckeditor5-ui/src/editorui/editorui.ts:352]: I see that you migrate it, thanks!

**Review by DawidKossowski — CHANGES_REQUESTED** [review]
I just found that there are missing tests for `editor-balloon` in the context of the menu bar.

We also moved some logic from `ui/utils` to `ui/editorui`. Shouldn't we also move some unit tests (like focus handling tests)?

**Review by DawidKossowski — APPROVED** [review]

**Pull request merged**